mqtt2kasa/log.py

```diff
@@ -8,31 +8,11 @@ def getLogger():
     return logging.getLogger("mqtt2kasa")
 
 
-def _log_handler_address(files=tuple()):
-    try:
-        return next(f for f in files if path.exists(f))
-    except StopIteration:
-        pass
-    raise Exception("Invalid files: %s" % ", ".join(files))
-
-
 def initLogger(testing=False):
     logger = getLogger()
     logger.setLevel(logging.INFO)
-    format = (
-        "%(asctime)s [mqtt2kasa] %(module)12s:%(lineno)-d %(levelname)-8s %(message)s"
-    )
-    formatter = logging.Formatter(format)
-
-    # Logs are normally configured here: /etc/rsyslog.d/*
-    logHandlerAddress = _log_handler_address(
-        ["/run/systemd/journal/syslog", "/var/run/syslog", "/device/log"]
-    )
-    syslog = SysLogHandler(address=logHandlerAddress, facility=SysLogHandler.LOG_DAEMON)
-    syslog.setFormatter(formatter)
-    logger.addHandler(syslog)
+    log_to_console()
```

Comment:
johnjones4  hm... looks like `_log_handler_address` is not working for you, right? I normally use that and wonder if there is a way to make it work for you w/out removing it completely.